core/src/test/scala/org/apache/spark/scheduler/cluster/mesos/Utils.scala

```diff
@@ -19,15 +19,19 @@ package org.apache.spark.scheduler.cluster.mesos
 
 import java.util.Collections
 
+import scala.collection.JavaConverters._
+
 import org.apache.mesos.Protos._
-import org.apache.mesos.Protos.Value.Scalar
+import org.apache.mesos.Protos.Value.{Range => MesosRange, Ranges, Scalar}
 import org.apache.mesos.SchedulerDriver
 import org.mockito.{ArgumentCaptor, Matchers}
 import org.mockito.Mockito._
-import scala.collection.JavaConverters._
 
 object Utils {
-  def createOffer(offerId: String, slaveId: String, mem: Int, cpu: Int): Offer = {
+  def createOffer(
```

Comment:
params should follow this style https://github.com/databricks/scala-style-guide#spacing-and-indentation